core/src/main/java/com/novoda/noplayer/PlayerState.java

```diff
@@ -75,6 +85,14 @@
      */
     long mediaDurationInMillis();
```

Comment:
So this is going to be the same as `contentDurationInMillis` when playing main content and changing when playing an ad, right? (same for `playheadPositionInMillis`?)

Reply:
Yea, which makes it annoying for clients to use because we can't guarantee the behaviour. Whereas the new methods will only talk about content.